test/test_library.py

```diff
@@ -585,13 +585,49 @@ def test_upper_case_variable(self):
         self._assert_dest(b'/base/THE TITLE')
 
     def test_title_case_variable(self):
-        self._setf(u'%title{$title}')
-        self._assert_dest(b'/base/The Title')
+        self._setf(u'%title{title}')
+        self._assert_dest(b'/base/Title')
```

Comment:
I'm not sure why this test changed? We should add a new one if we just want to test non-variable title-casing.